src/app/api/sendpulse/route.ts

```
@@ -0,0 +1,185 @@
+import { NextRequest, NextResponse } from 'next/server'
+import { sendPulseClient } from '@/lib/sendpulse-client'
+
+// Get SendPulse account info and test connection
+export async function GET(request: NextRequest) {
+  try {
+    const { searchParams } = new URL(request.url)
+    const action = searchParams.get('action')
+
+    switch (action) {
+      case 'test':
+        const isConnected = await sendPulseClient.testConnection()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)